src/store/student-store.js

```diff
@@ -18,13 +18,8 @@ class StudentStore extends Store{
 const store = new StudentStore();
 
 store.registerHandler('STUDENT_FOUND', data => {
-    student = {
-        //NOTE: this data is tentative
-        id : data.id,
-        name: data.name,
-        items: data.items,
-        hasOverdueItem: store.hasOverdueItems(data.items)
-    };
+    student = data;
+    student.hasOverdueItem = store.hasOverdueItems(data.items);
```

Comment:
You wrote this line.

`hasOverdueItems` is fine. Assigning the result to a variable like this and then reusing that variable is not.